src/components/pages/DataVisualizations/GraphWrapper.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import CitizenshipMapAll from './Graphs/CitizenshipMapAll';
+import CitizenshipMapSingleOffice from './Graphs/CitizenshipMapSingleOffice';
+import TimeSeriesAll from './Graphs/TimeSeriesAll';
+import OfficeHeatMap from './Graphs/OfficeHeatMap';
+import TimeSeriesSingleOffice from './Graphs/TimeSeriesSingleOffice';
+import YearLimitsSelect from './YearLimitsSelect';
+import ViewSelect from './ViewSelect';
+import axios from 'axios';
+import { resetVisualizationQuery } from '../../../state/actionCreators';
+import test_data from '../../../data/test_data.json';
+import { colors } from '../../../styles/data_vis_colors';
+
+const { background_color } = colors;
+
+function GraphWrapper(props) {
+  const { set_view, dispatch } = props;
+  const { office, view } = useParams();
+  let map_to_render;
+  if (!office) {
+    switch (view) {
+      case 'time-series':
+        map_to_render = <TimeSeriesAll />;
+        break;
+      case 'office-heat-map':
+        map_to_render = <OfficeHeatMap />;
+        break;
+      case 'citizenship':
+        map_to_render = <CitizenshipMapAll />;
+        break;
+      default:
+        break;
+    }
+  } else {
+    switch (view) {
+      case 'time-series':
+        map_to_render = <TimeSeriesSingleOffice office={office} />;
+        break;
+      case 'citizenship':
+        map_to_render = <CitizenshipMapSingleOffice office={office} />;
+        break;
+      default:
+        break;
+    }
+  }
+  function updateStateWithNewData(view, office, stateSettingCallback) {
+    if (office === 'all' || !office) {
+      axios
+        .get('http://localhost:3000')
+        .then(result => {
+          stateSettingCallback(view, office, test_data);
+        })
+        .catch(err => {
+          console.error(err);
+        });
+    } else {
+      axios
+        .get('http://localhost:3000')
+        .then(result => {
+          stateSettingCallback(view, office, test_data);
+        })
+        .catch(err => {
+          console.error(err);
+        });
+    }
+  }
+  const clearQuery = (view, office) => {
+    dispatch(resetVisualizationQuery(view, office));
+  };
+  return (
+    <div
+      className="map-wrapper-container"
+      style={{
+        display: 'flex',
+        alignItems: 'flex-start',
+        justifyContent: 'center',
+        minHeight: '50px',
+        backgroundColor: background_color,
+      }}
+    >
+      {map_to_render}
+      <div
+        className="user-input-sidebar-container"
+        style={{
```

Comment:
Could we move the styles to a file designated for this file?

Reply:
I would be happy to extract styles and organize them in external files; it feels like something that would maybe be good for another ticket?

Reply:
Let's make an GitHub issue for this. And then with John's approval, we can transfer it into a ticket.